src/amo/components/EditableCollectionAddon/styles.scss

```diff
@@ -4,11 +4,48 @@
 @import '~core/css/inc/vars';
 @import '~ui/css/vars';
 
-.EditableCollectionAddon {
-  display: grid;
-  grid-gap: 24px;
-  grid-template-columns: 32px minmax(80px, auto) 32px 64px;
-  padding: 12px;
+@include respond-to(large) {
+  .EditableCollectionAddon {
+    display: grid;
+    grid-gap: 24px;
+    grid-template-columns: minmax(112px, auto) min-content;
+  }
+}
+
+.EditableCollectionAddon-buttons,
+.EditableCollectionAddon-details {
+  align-items: center;
+  display: flex;
+
+  @include respond-to(large) {
+    display: grid;
+    grid-gap: 24px;
+    padding: 0;
+  }
+}
+
+.EditableCollectionAddon-details {
+  grid-template-columns: 32px minmax(80px, auto);
+}
+
+.EditableCollectionAddon-buttons {
+  grid-template-columns: min-content min-content;
+}
+
+.EditableCollectionAddon > div {
```

Comment:
It's also `.EditableCollectionAddon-buttons`, so I did this to avoid duplicating it in both places. I think originally it also matched more than those two, but my latest redesign still makes it match two elements.

Reply:
ah, then you could probably move it to the block line 15-16?

Reply:
Actually, it also applies to `.EditableCollectionAddon-notes`, which I noticed when I moved it, but I just added it there as well.